src/rust/engine/src/tasks.rs

```diff
@@ -86,16 +86,6 @@ impl Tasks {
     );
   }
 
-  // TODO: Only exists in order to support the `Snapshots` singleton replacement in `context.rs`:
-  // Fix by porting isolated processes to rust:
```

Comment:
This is probably blocked on landing #4397, because the python code currently cheats, consumes this object, and goes looking for tar files.